### PR TITLE
Crom my contribution

### PR DESCRIPTION
**Pull request by csrordzhn**
### Description

Addiing a new term entry for trojan under the malware section, #2688 
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cybersecurity/concepts/malware/terms/trojan/trojan.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Trojan' 
+Description: 'A trojan is a type of malware that downloads onto a computer disguised as a legitimate program.'
```
Comment (dakshdeepHERE) [inline content/cybersecurity/concepts/malware/terms/trojan/trojan.md:3]:
```suggestion
Description: 'A type of malicious software that appears harmless but secretly carries out harmful actions on a computer system'
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I change from my side and I have one more comment to make in overall entry

**Comment by dakshdeepHERE** [comment]
Your entry is very similar and matches word to word from these websites [**1**](https://www.fortinet.com/resources/cyberglossary/trojan-horse-virus#:~:text=Trojans%20can%20also%20attack%20and,using%20it%20to%20launch%20cyberattacks.) [**2**](https://www.fortinet.com/resources/cyberglossary/trojan-horse-virus) 
Screenshot 2023-07-13 at 2 55 47 PM
Screenshot 2023-07-13 at 2 57 05 PM
You can try to rephrase the content so it doesn't flag it as plagiarised content to fix this issue 🙂.

**Comment by SSwiniarski** [comment]
As @dakshdeepHERE mentioned, this PR entry is copied from [here](https://www.fortinet.com/resources/cyberglossary/trojan-horse-virus).
We take plagiarism seriously for Codecademy Docs and for Codecademy content in general because a single case of plagiarism could jeopardize the credibility of all other 650+ entries as well as our greater catalog.
Here's the Codecademy Docs [Code of Conduct](https://github.com/yangc95/docs/blob/main/.github/CODE_OF_CONDUCT.md) on GitHub.
Since this is your first occurrence, I am issuing a warning.
We truly appreciate your help with your contribution so far and we don't want to lose you temporarily or permanently. But as we attempt to build the best coding documentation on the web, it's a long-term game and we want to focus on quality, not quantity!
We hope you understand why we are taking this very seriously.

Needless to say this entry cannot be considered until it is rewritten.

**Comment by csrordzhn** [comment]
Got it. I apologize for this and thanks for the warning. I will work on this issue this week.

**Comment by SSwiniarski** [comment]
@csrordzhn Because of the inactivity on this I am closing this PR.

**Pull request closed without merging**